docs/cody/clients/model-configuration.mdx

```diff
@@ -0,0 +1,520 @@
+# LLM Model Configuration
+
+<p className="subtitle">This guide will walk you through the steps to customize the LLM models available from your Sourcegraph Enterprise instance.</p>
+
+<Callout type="note">For Sourcegraph Cloud customers, configuring the available LLM models requires contacting Sourcegraph's tech support.</Callout>
+
+Starting with Sourcegraph 5.5.4+ Sourcegraph Enterprise supports the ability to choose between different LLM models, allowing developers to use the best model for their current needs.
+
+This is accomplished exposing much more flexible configuration options for Cody when using Sourcegraph Enterprise. The newer style of configuration is described next. However, you can still use the [Older style "Completions" Configuration](#Older-style-Completions-Configuration).
```

Comment:
This is okay for Aug 7th, but I suspect that we will want to have better 'customer facing' names in Sept

cc: @kelvinyap